linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/tests/pam_tally2_deny_missing.fail.sh

```diff
@@ -8,6 +8,18 @@ account required                        pam_tally2.so
 account	required			pam_permit.so
 CAPTC
 
+{{% if product in ["sle12","sle15"] %}}
+
+cat >/etc/pam.d/login <<CAPTDM
```

Comment:
The only change between this jinja2 conditional is the destination file. You can include only the destination file in the conditional and avoid many repeated lines.